Xamarin.Forms.Core/PlatformConfiguration/iOSSpecific/StatusBarTextColorMode.cs

```diff
@@ -0,0 +1,8 @@
+namespace Xamarin.Forms.PlatformConfiguration.iOSSpecific
+{
+	public enum StatusBarTextColorMode
+	{
+		MatchNavigationBarTextLuminosity,
+		DoNotAdjust
```

Comment:
I'm not particularly happy with this naming. I want it to be `Default`, but the Forms "default" is to adjust the text color. If anyone has a better name, please feel free to suggest it.